### ISSUE TITLE
Reflex vasodepressor reduction to palpitations

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10249]
**Preferred term label:**
Reflex vasodepressor reduction to palpitations

**Synonyms**
Neurocardiogenic; breath-holding, cough, micturition, defecation, carotid sinus pressure, valsalva, or orthostasis

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
supraventricular tachycardia

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
is this referring to effects induced by carotid sinus massage? I am not sure I understand the intended meaning

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK436016/
In patients who can terminate their palpitations with a Valsalva maneuver, this is thought to indicate possibly a supraventricular tachycardia.

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000035
- label: Palpitations terminated by Valsava maneuver
- parents: [Ameliorated by (HP:0025254)](https://hpo.jax.org/app/browse/term/HP:0025254)
- pmid: PMID:28613787
- def: Cessation of palpitations can be induced by Valsava maneuver.
- comment: In patients who can terminate their palpitations with a Valsalva maneuver, this is thought to indicate possibly a supraventricular tachycardia.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10249
- synonyms:
